lms/templates/instructor/instructor_dashboard_2/special_exams.html

```diff
@@ -10,22 +10,22 @@ <h6 class = "hd hd-6 escalation-email-container">${_('Proctortrack Escalation Em
   % endif
   <div id="proctoring-accordion">
     <div class="wrap">
-      <h3 class="hd hd-3">${_('Allowance Section')}</h3>
+      <h3 class="hd hd-3"><div class="special-exam-header">${_('Allowance Section')}</div></h3>
```

Comment:
You can just add the class to the h3 like `<h3 class="hd hd-3 special-exam-header">`. Then you don't have the wrap the text around `div`